storage/database/db_manager.go

```diff
@@ -593,8 +593,6 @@ func (dbm *databaseManager) setStateTrieMigrationStatus(blockNum uint64) {
 	if err := miscDB.Put(migrationStatusKey, common.Int64ToByteBigEndian(blockNum)); err != nil {
 		logger.Crit("Failed to set state trie migration status", "err", err)
 	}
-	dbm.lockInMigration.Lock()
-	defer dbm.lockInMigration.Unlock()
```

Comment:
Why are these lines removed?

Reply:
To expand the scope of the lock, I moved this to callee function.
See line 641. :)